src/lib.rs

```diff
@@ -72,14 +72,22 @@ impl Project {
         // parse Cargo.toml
         let toml = root.join("Cargo.toml");
         let cargo_config = Path::new(".cargo").join("config");
+        let cargo_config_toml = Path::new(".cargo").join("config.toml");
```

Comment:
This might be a bit clearer:

```suggestion
        let cargo_config_toml = cargo_config.with_extension("toml");
```